### DISCUSSION TITLE
This is a wfly et16s transmitter with an betafpv express module. Will this work?

### DISCUSSION THREAD

**Discussion by DethRattles** [ExpressLRS/ExpressLRS#2527] [Q&A]
I was wondering if anyone can make sense of this all. I was able to bind the newbeedrone hummingbird with this even though this radio does not have a sd card or lua script. I have no stick movements in betaflight and now did something to where the module says mismatch. Anyways, i had a helluva time upgrading this radio from 1.30v to 1.62 version with expresslrs. Wfly says the radio works with modules like blacksheep crossfire and expresslrs. I want to show you what my radio shows and the module and tell me if this radio has the lua scriot built in and if i'm able to get the sticks working and get rid of this mismatch? In betaflight i have to hit the bind button through that to get the quad to stop spinning on the reciever page in betaflight. Anyways,,binding through the radio gets me a solid green light which newbeedrone says its bound when that happens. I'm uploading a video but its too big so i've uploaded it to a website here https://files.catbox.moe/5e4lzm.mp4

**Comment by kautionfpv** [comment]
I am not seeing where it says mismatch? You can try turning your packet rate down to 250hz and see if that helps. Also I would turn your transmit power way down, max 250mw, no need for 500 unless you are doing some serious long range flying. I would also question the "built in" LUA script. Do you have a binding phrase setup on your BetaFPV module? And if so did you make sure this was the same in Betaflight?

**Reply by DethRattles** [reply]
Thanks for responding Kaution!, It occasionally says mismatch on the module. I'm new to expresslrs and modules. . I reached out to Wfly and they contacted me and said that the module should work, then after it didn't said they recommend their wfly module and receivers. I got the betafpv module in a classified add and haven't checked if it has a binding phrase already in it but betaflight's phrase was blank. The quad did bind though, i had a steady green light on the quad rather than siren lights. I just couldn't get betaflight to recognize the controller. I finally got a radiomaster elrs pocket and it took me 3 hours to bind that with watching youtube vids and learning more/less about elrs. The wizard gave me an arithmetic error after i'd try to save,,then i finally put the model on crsf and it bound. I read somewhere that express goes through crsf during the wfly experiment. I really want to try to get the wfly to work but theres no telling if they have a built in lua script, it could be their own but they can work with blacksheep so i figured there's a chance. 

 Why would it give me a steady green light on my newbeedrone hummingbird? If it had a problem with the phrase i would think it wouldnt bind at all. I need to learn about packet size, At first thought bigger was better. I also couldnt figure out how to get to menu's in the module until much later into the hours i was working on it. The instructions were wrong. The wfly has a 150 page manual but nothing about getting modules setup. Again i want to thank you for reaching out.
